### PR TITLE
SnPM updates

### PR DESCRIPTION
**Pull request by TomMaullin**
Hi @nicholst 

This is a quick PR to remove all reference of the `SnPM updates` page as requested via email. Please let me know if you have any feedback/are happy for this to be merged.

(Edit: I have just realized I forked this from my master branch which already had merged some of the edits mentioned in PR #61 . Apologies for this, because of this it would probably be best to review and merge PR #61 before this PR.)

### CODE REVIEW

File: Software/SnPM13/distribution.html [hunk 1]
```diff
@@ -41,21 +41,20 @@ <h2><a name="help">Support</a></h2>
...
-
 <!-- <p>None found yet!</p> -->
 <!--<p>Installation is relatively straightforward. Download the gzipped tar file and unzip it in a location where you want the snpm3 directory. Then, add the full pathname of this directory to your MATLABPATH. (You can check this by typing <code>which spm</code> in MatLab and examining the path, or try <code>spm ver</code>.) </p>-->
-<!--<p>Please report bugs to &lt;<a href="mailto:snpm-support at googlegroups.com.">snpm-support at googlegroups.com.</a>&gt;. Peculiarities and general queries should also be raised on the <a href="http://www.fil.ion.ucl.ac.uk/spm/help/">SPM <em>e</em>mail discussion list</a>. We will report patches to the list and make updates available if necessary. -->
+<!--<p>Please report bugs to &lt;<a href="mailto:snpm-support at googlegroups.com.">snpm-support at googlegroups.com.</a>&gt;. Peculiarities and general queries should also be raised on the <a href="http://www.fil.ion.ucl.ac.uk/spm/help/">SPM <em>e</em>mail discussion list</a>.  -->
```
Comment (nicholst) [inline Software/SnPM13/distribution.html]: Same here... delete reference to SPM list
> . Peculiarities and general queries should also be raised on the <a href="http://www.fil.ion.ucl.ac.uk/spm/help/">SPM <em>e</em>mail discussion list

File: Software/SnPM13/distribution.html [hunk 2]
```diff
@@ -41,21 +41,20 @@ <h2><a name="help">Support</a></h2>
...
 <h2><a name="help">Requirements</a></h2>
 
 <p>The SnPM toolbox is a small suite of MatLab functions and M-files that integrate with <a href="http://www.fil.ion.ucl.ac.uk/spm/">SPM</a> routines. Thus, to use SnPM13 you must have installed SPM8 or SPM12b, to use SnPM8 you must have installed SPM8, to use SnPM5 you must have already installed SPM5, and to use SnPM3 you must have already installed SPM2. See the <a href="http://www.fil.ion.ucl.ac.uk/spm/distrib.html">SPM distribution page</a> for SPM hardware &amp; software requirements, downloading &amp; installation instructions, and conditions of distribution. You should be familiar with SPM before using SnPM. </p>
 
 <p>For step by step instructions on downloading and installing SPM distributions, refer to the <a href="http://www.fil.ion.ucl.ac.uk/spm/distrib.html# Download">SPM distribution page</a>. </p>
 
+<h2><a name="update">Updates</a></h2>
+
+<p>To check for updates, open <code>SnPM</code> and type <code>SnPM_update()</code>. This function will tell you whether your current version of <code>SnPM</code> is up to date and where to download a newer version if not. (Note: This function requires an internet connection).</p>
```
Comment (nicholst) [inline Software/SnPM13/distribution.html]: Open/close parents are not neccssary (and may create an error). Make this
> ... type `SnPM_update`. This...

File: _site/Software/SnPM13/distribution.html [hunk 3]
```diff
@@ -115,21 +115,20 @@ <h2><a name="help">Support</a></h2>
 
 <p><strong>Please email the authors <a href="mailto:snpm-support@googlegroups.com">snpm-support at googlegroups.com</a> </strong>with SnPM issues and idiosyncrasies, but also please CC the <a href="http://www.fil.ion.ucl.ac.uk/spm/help/">SPM <em>e</em>mail discussion list</a>, at &lt;<a href="mailto:spm@jiscmail.ac.uk">spm@jiscmail.ac.uk</a>&gt; so other SPM users can learn from the questions. </p>
```
Comment (nicholst) [inline _site/Software/SnPM13/distribution.html]: Again, please delete reference to SPM:
> , but also please CC the <a href="http://www.fil.ion.ucl.ac.uk/spm/help/">SPM <em>e</em>mail discussion list</a>, at &lt;<a href="mailto:spm@jiscmail.ac.uk">spm@jiscmail.ac.uk</a>&gt; so other SPM users can learn from the questions

File: _site/Software/SnPM13/distribution.html [hunk 4]
```diff
@@ -115,21 +115,20 @@ <h2><a name="help">Support</a></h2>
...
-
 <!-- <p>None found yet!</p> -->
 <!--<p>Installation is relatively straightforward. Download the gzipped tar file and unzip it in a location where you want the snpm3 directory. Then, add the full pathname of this directory to your MATLABPATH. (You can check this by typing <code>which spm</code> in MatLab and examining the path, or try <code>spm ver</code>.) </p>-->
-<!--<p>Please report bugs to &lt;<a href="mailto:snpm-support at googlegroups.com.">snpm-support at googlegroups.com.</a>&gt;. Peculiarities and general queries should also be raised on the <a href="http://www.fil.ion.ucl.ac.uk/spm/help/">SPM <em>e</em>mail discussion list</a>. We will report patches to the list and make updates available if necessary. -->
+<!--<p>Please report bugs to &lt;<a href="mailto:snpm-support at googlegroups.com.">snpm-support at googlegroups.com.</a>&gt;. Peculiarities and general queries should also be raised on the <a href="http://www.fil.ion.ucl.ac.uk/spm/help/">SPM <em>e</em>mail discussion list</a>.  -->
```
Comment (nicholst) [inline _site/Software/SnPM13/distribution.html]: Here and 118-120 in fact are all commented-out... let's just delete, OK?

File: _site/Software/SnPM13/distribution.html [hunk 5]
```diff
@@ -115,21 +115,20 @@ <h2><a name="help">Support</a></h2>
...
 <h2><a name="help">Requirements</a></h2>
 
 <p>The SnPM toolbox is a small suite of MatLab functions and M-files that integrate with <a href="http://www.fil.ion.ucl.ac.uk/spm/">SPM</a> routines. Thus, to use SnPM13 you must have installed SPM8 or SPM12b, to use SnPM8 you must have installed SPM8, to use SnPM5 you must have already installed SPM5, and to use SnPM3 you must have already installed SPM2. See the <a href="http://www.fil.ion.ucl.ac.uk/spm/distrib.html">SPM distribution page</a> for SPM hardware &amp; software requirements, downloading &amp; installation instructions, and conditions of distribution. You should be familiar with SPM before using SnPM. </p>
 
 <p>For step by step instructions on downloading and installing SPM distributions, refer to the <a href="http://www.fil.ion.ucl.ac.uk/spm/distrib.html# Download">SPM distribution page</a>. </p>
 
+<h2><a name="update">Updates</a></h2>
+
+<p>To check for updates, open <code>SnPM</code> and type <code>SnPM_update()</code>. This function will tell you whether your current version of <code>SnPM</code> is up to date and where to download a newer version if not. (Note: This function requires an internet connection).</p>
```
Comment (nicholst) [inline _site/Software/SnPM13/distribution.html]: Change to...
> ... type `SnPM_update`. This...

File: _site/Software/SnPM13/index.html [hunk 6]
```diff
@@ -103,7 +103,7 @@ <h2><a name="news">Introduction</a> </h2>
    </dt> 
    <dd>
     Citation of the SnPM software can be made with reference to this URL 
-    <a href="http://nisox.org/Software/SnPM13/"><tt>http://nisox.org/Software/SnPM13/</tt></a>; please also note the version (i.e. SnPM13) and the date that you last checked for updates. Concepts implemented in the SnPM software are best described in the Nichols &amp; Holmes (2001) paper; see 
+    <a href="http://nisox.org/Software/SnPM13/"><tt>http://nisox.org/Software/SnPM13/</tt></a>; please also note the version (i.e. SnPM13) in any citation. Concepts implemented in the SnPM software are best described in the Nichols &amp; Holmes (2001) paper; see
```
Comment (nicholst) [inline _site/Software/SnPM13/index.html]: Here, and all recommendations for Citation (in other files) can we issue better best practice... i.e. give the *exact* version.  Something like:

> please also note the exact version (e.g. SnPM13.1.06) to maximise reproducibility.  The full version number can be found from the `snpm('ver')` command.

(check that `snpm('ver')`  works!)

**Comment by nicholst** [comment]
See my various comments... once these are addressed we're good to go!

**Comment by TomMaullin** [comment]
Hi @nicholst ,

All feedback has now been addressed, please let me know if you are happy to merge!

**Pull request merged**